dc_kinesis_clickhouse_transfer/README.md

```diff
@@ -0,0 +1,236 @@
+
+```shell
+terraform apply \
+  -var="stream_name=YOUR_STREAM_NAME" \
+  -var="enable_transfer=true" \
+  -var="project_id=YOUR_PROJECT_ID" \
+  -var="aws_profile=YOUR_AWS_PROFILE" \
+  -var="federation_id=YOUR_DOUBLECLOUD_FEDERATION_ID" \
+  -var="aws_access_key_id=AWS_ACCESS_KEY_ID" \
+  -var="aws_region_id=AWS_REGION_ID" \
+  -var="endpoint=DOUBLECLOUD_REST_API_ENDPOINT" \
+  -var="federation_endpoint=DOUBLECLOUD_gRPC_ENDPOINT" \
+  -var="aws_secret_access_key=AWS_SECRET_ACCESS_KEY" 
+```
+
+This option will create a transfer, from `./transfer.tf`
+
+Once Terraform is applied, you will see the following
+
+```shell
+Apply complete! Resources: 5 added, 0 changed, 0 destroyed.
+
+Outputs:
+
+clickhouse_connection = {
+  "host" = "CLICKHOUSE_HOST"
+  "https_port" = 8443
+  "password" = "CLICKHOUSE_PASSWORD"
+  "tcp_port_secure" = 9440
+  "user" = "admin"
+}
+```
+
+The main thing on transfer configuration - parser rule:
+
+[transfer.tf#L10-L67](./transfer.tf#L10-L67)
+```hcl
+      parser {
+        json {
+          schema {
+            fields {
+              field {
+                name     = "Topic"
+                type     = "string"
+                key      = true
+                required = true
+              }
+              field {
+                name     = "Partition"
+                type     = "uint32"
+                key      = true
+                required = true
+              }
+              field {
+                name     = "Offset"
+                type     = "uint32"
+                key      = true
+                required = true
+              }
+              field {
+                name     = "Timestamp"
+                type     = "datetime"
+                key      = false
+                required = false
+              }
+              field {
+                name     = "Headers"
+                type     = "string"
+                key      = false
+                required = false
+              }
+              field {
+                name     = "Value"
+                type     = "string"
+                key      = false
+                required = false
+              }
+              field {
+                name     = "Transfer_commit_time"
+                type     = "uint64"
+                key      = false
+                required = false
+              }
+              field {
+                name     = "Transfer_delete_time"
+                type     = "uint64"
+                key      = false
+                required = false
+              }
+            }
+          }
+          null_keys_allowed = false
+          add_rest_column   = true
+        }
+      }
+```
+
+After applying, it will create transfer:
+
+![transfer.png](./assets/transfer.png)
+
+This transfer automatically creates a table called: `${STREAM_NAME}_unparsed`, the name was inferred from the Kinesis stream name, and it delivers rows into it:
+
+```sql
+SELECT count(*)
+FROM  ${STREAM_NAME}_unparsed LIMIT 100
```

Comment:
_unparsed tables is usually exists when your data is not parsed, it's good to show here both: parsed and unparsed data